### ISSUE TITLE
moment-timezone does not support abbreviation for some timezones - Asia/Kamchatka timezone

### ISSUE THREAD

**Issue by garishsamuel** [moment/moment-timezone#942]
**Version:** 0.5.33

**Issue description:**
moment.tz() method not returning the timezone abbreviation for some timezones - Eg: Asia/Kamchatka timezone.
It is returning **+12** instead of **PETT**.

date = moment().tz('Asia/Kamchatka').format('DD-MMM-YYYY h:mm A z');

**Expected Result:** 26-May-2021 9:46 PM **PETT**
**Actual Result:** 26-May-2021 9:46 PM **+12**

**Stackblitz Example:** https://stackblitz.com/edit/moment-js-5lmc4h

This issue is exists for the below timezones:

1. Asia/Kamchatka
2. Pacific/Niue
3. Pacific/Nauru

**Issue Screenshot:**
![image](https://user-images.githubusercontent.com/35136769/119645650-e3417d80-be3b-11eb-9605-acf415542611.png)

**Comment by Emeka-Ugbanu-hub** [comment]
i would love to help with this

**Comment by ichernev** [comment]
You can check https://github.com/moment/moment-timezone/issues/723

TL;DR: It's a feature
